src/Symfony/Component/HtmlSanitizer/Reference/W3CReference.php

```diff
@@ -0,0 +1,383 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HtmlSanitizer\Reference;
+
+/**
+ * Stores reference data from the W3C Sanitizer API standard.
+ * {@see https://wicg.github.io/sanitizer-api/#default-configuration}.
+ *
+ * @author Titouan Galopin <galopintitouan@gmail.com>
+ *
+ * @internal
+ */
+final class W3CReference
```

Comment:
how did you generate this class? how will we update it if the w3c updates it?

Reply:
I added a test case to ensure it'll stay in sync with the standard.